OpenSim/Examples/ExampleHopperDevice/buildHopperModel.cpp

```diff
@@ -88,15 +88,16 @@ Model buildHopper() {
 
     // Set the coordinate names and default values. Note that we need "auto&"
     // here so that we get a reference to the CoordinateSet rather than a copy.
-    auto& sliderCoord = sliderToGround->upd_CoordinateSet()[0];
+    auto& sliderCoord =
```

Comment:
> Maybe the PR you fixed it in has not been merged yet.

Ah, you are right. I slipped it into #1214 because I wasn't expecting that to be controversial.